src/components/structures/GroupView.js

```diff
@@ -197,6 +199,10 @@ export default React.createClass({
         }
     },
 
+    _collectChangeAvatar: function(node) {
+        this._changeAvatarComponent = node;
+    },
+
     _loadGroupFromServer: function(groupId) {
         MatrixClientPeg.get().getGroupSummary(groupId).done((res) => {
```

Comment:
We set the summary to null after saving which will cause it to show a spinner until it becomes non-null